sklearn/linear_model/_bayes.py

```diff
@@ -509,6 +553,11 @@ class ARDRegression(RegressorMixin, LinearModel):
 
         .. versionadded:: 1.0
 
+    n_iter_ : int
```

Comment:
Have I included this attribute correctly? It previously wasn't there in `ARDRegression` but is required for estimators that have the `max_iter` attribute.

Reply:
Let's put it under `scores_` to be consistent with `BayesRidge`.